pkg/agent/controller/egress/egress_controller.go

```diff
@@ -974,3 +1013,27 @@ func (c *EgressController) GetEgress(ns, podName string) (string, string, error)
 func isEgressSchedulable(egress *crdv1b1.Egress) bool {
 	return egress.Spec.EgressIP != "" && egress.Spec.ExternalIPPool != ""
 }
+
+// compareEgressStatus compares two Egress Statuses, ignoring LastTransitionTime and conditions other than IPAssigned, returns true if they are equal.
+func compareEgressStatus(currentStatus, desiredStatus crdv1b1.EgressStatus) bool {
+	if currentStatus.EgressIP == desiredStatus.EgressIP && currentStatus.EgressNode == desiredStatus.EgressNode {
+		return true
+	}
```

Comment:
```suggestion
	if currentStatus.EgressIP != desiredStatus.EgressIP || currentStatus.EgressNode != desiredStatus.EgressNode {
		return false
	}
```
Didn't the unit test fail?

Reply:
Apologies, I was editing on macos and didn't run the unit test.